pkg/drain/cordon.go

```diff
@@ -31,32 +31,21 @@ import (
 
 // CordonHelper wraps functionality to cordon/uncordon nodes
 type CordonHelper struct {
-	node   *corev1.Node
-	status DesiredCordonStatus
-}
-
-type DesiredCordonStatus string
-
-const (
-	CordonNode   DesiredCordonStatus = "cordon"
-	UncordonNode DesiredCordonStatus = "uncordon"
-)
-
-func (n DesiredCordonStatus) String() string {
-	return string(n)
+	node    *corev1.Node
+	desired bool
```

Comment:
I think bools are usually terrible for communicating what they mean. I actually prefer the old constants. Perhaps rename `desired` to `cordon`?

Reply:
CordonHelper either cordons or uncordons nodes. The old constants were redundant to define that boolean state and also required more code. 
The field is named `desired` because it reflects the type's default behaviour (`CordonHelper`). I'm not opposed to renaming it to `cordon` though as I think it does make the intent more explicit.

Reply:
Some of the initial code was borrowed from `kubectl`, which also uses `CordonHelper.desired`, so I think we should keep that name.

Reply:
It was more code but it was more clear. less code is not always better :)

What do you think about `cordoned`?

Reply:
Hm, althought `cordoned` could be understood as  the actual state. Yeah, leave it as desired if you feel strongly against what we had before.

Reply:
>It was more code but it was more clear. less code is not always better :)

I don't think it was more clear :) It was a boolean state